### PR TITLE
Design Teardown

### PR DESCRIPTION
**Pull request by BigWizzo**
#HTML and CSS: Design Teardown

* On the branch more-content we created the entire project that includes the styling and the skeleton of the HTML.
* We applied flex boxes, floats, and CSS grids.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,30 @@
...
-# Project Name
+# Design Teardown
 
-> One paragraph statement about the project.
+Design Teardown Heatmap
 
-![screenshot](./app_screenshot.png)
+![screenshot](./app_screenshot.png/)
 
 Additional description about the project and its features.
```
Comment (SafaErden) [inline README.md]: - [x] Please describe your project briefly.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+      <!-- middle section -->
+      <div class="middle-comntainer flex-row fourth-color">
+        <div class="middle-images article-color">
+          <div class="signin"></div>
+          <div class="shopify-container flex-col">
+            <div class="shopiitems flex-col-aic">
+              <p class="person-para color-black"></p>
+              <p class="person-para-last color-black"></p>
+            </div>
+            <!-- <div class="shopiitems"></div> -->
```
Comment (SafaErden) [inline index.html]: - [x] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: index.html [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+            <span class="tag-lw-span color-black"></span>
+          </div>
+          <div class="top-tag flex-row">
+            <span class="tag-lw-span color-black"></span>
+            <span class="tag-lw-span color-black"></span>
+            <span class="tag-lw-span color-black"></span>
+          </div>
+        </div>
+        <div class="foot-box color-black"></div>
+      </div>
```
Comment (SafaErden) [inline index.html]: - [x] The footer contains centered links, so please convert these span tags into the proper HTML tag to make them links.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,499 @@
...
+      <div class="griditems5 d-grid fifth-color">Events</div>
+      <div class="griditems6 d-grid fifth-color">Membership</div>
+      <div class="griditems7 d-grid fifth-color">More</div>
+      <div class="griditems8 boxes">
+        <input type="search" placeholder="Search articles" />
+      </div>
+    </nav>
+
+    <!-- featured articles -->
+    <div class="featured flex-col">
```
Comment (SafaErden) [inline index.html]: - [OPTIONAL] You can consider the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

File: index.html [hunk 5]
```diff
@@ -0,0 +1,499 @@
...
+        <div class="news-griditem">
+          <article>
+            <div class="content"></div>
+          </article>
+        </div>
+      </div>
+    </div>
+
+    <!-- footer section starts here -->
+    <div class="footer flex-col-aic fourth-color">
```
Comment (SafaErden) [inline index.html]: - [OPTIONAL] You can consider the use of the [`<footer>`](https://www.w3schools.com/TagS/tag_footer.asp) tag for the footer

**Review by SafaErden — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️

**Hi TEAM, your project is awesome. 👐 However, we need a little fix before we can proceed. Kindly find comments below**
##

**REQUIRED CHANGES**

- [x] There is an empty space on the right. Since you are allowed to create your pages using a width of **1440px**, there is no problem with the content width but for larger screens, you should center your content or make your content cover the whole width. Please kindly fix this issue.

image

- [1] The three article section contains 3 articles, again there is an extra space for the fourth article.

image

- [2] Also the one article section has extra space on the right. 

image

- [x] Since your implemented text colors are dark, they are not visible on the dark background. Please make your text white to make them visible.

image

- [x] All the elements and element's backgrounds and font colors should be grayscale color (black, white or any shade of gray), this rule also applies to images if the students decide to add any ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/grayscale_requirement.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/design-teardown))

Please kindly remove the colorful borders from your content.

image

- [x] Make sure that basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices) ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright). You repeat the same CSS rules in your stylesheet (display: flex, justify-content: center, etc.)

Please kindly apply DRY CSS rules to your stylesheet.

- [x] Additional required changes are listed below, please kindly fix them.

##

**NOTES**

Friendly reminder: Remember it's important to have a professional repo so you can increase your job opportunities. 

- So it is important to create a descriptive README file. In your README you should explain the project briefly. At least you can talk about that the project is the heatmap of the smashing magazine website.

- Also, the PR message should give a brief description of the project.

Please take a step back and read the lesson here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/. And pay more attention to these issues on your upcoming projects.

##

**SUMMARY**

As I mentioned above, your project is almost done. You have some minor issues to solve. Please be an optimist and do not forget that the main purpose is to learn and practice. Do not let the requested changes discourage you 👍

##

*Kindly make the changes and submit another code review when you are ready*



Questions or comments? Reach me on [Twitter](https://twitter.com/safaerden) or [LinkedIn](https://www.linkedin.com/in/safaerden/)

Happy coding_ 👨‍💻 

**Safa ERDEN**

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

### Hello @BigWizzo and @cvilla714 :wave: 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**